src/workerd/io/trace-stream.c++

```diff
@@ -651,7 +651,7 @@ class TailStreamTarget final: public rpc::TailStreamTarget::Server {
   KJ_DISALLOW_COPY_AND_MOVE(TailStreamTarget);
   ~TailStreamTarget() {
     if (doneFulfiller->isWaiting()) {
-      doneFulfiller->reject(KJ_EXCEPTION(DISCONNECTED, "Tail stream session canceled."));
+      doneFulfiller->reject(KJ_EXCEPTION(DISCONNECTED, "Streaming tail session canceled."));
```

Comment:
This is updated to so that the message matches the other stream cancellation warnings (the genuine one in TailStreamCustomEventImpl::sendRpc() and the misleading one being removed here) so that we can more easily detect stream cancellations in downstream regression tests.